### PR TITLE
Anomaly#10232-Create a new entry in menu for Analytic journal types

### PR DESCRIPTION
**Pull request by fno-axelor**

### CODE REVIEW

File: axelor-account/src/main/resources/views/Menu.xml [hunk 1]
```diff
@@ -398,6 +398,15 @@
 		<view type="grid" name="analytic-journal-grid"/>
 		<view type="form" name="analytic-journal-form"/>
 	</action-view>
+	
+	<menuitem name="account-conf-analytic-journal-type" parent="account-conf-analytic"
+		title="Analytic journals type" action="account.conf.analytic.journal.type" />
+
+	<action-view name="account.conf.analytic.journal.type" title="Analytic journals type"
```
Comment (gdu-axelor) [inline axelor-account/src/main/resources/views/Menu.xml]: Analytic journal types ?

Comment (fno-axelor) [reply axelor-account/src/main/resources/views/Menu.xml]: oups! ok

Comment (gdu-axelor) [reply axelor-account/src/main/resources/views/Menu.xml]: Same issue on the action-view title

**Comment by gdu-axelor** [comment]
Changelog  ?

**Pull request merged**